TransIT.API/Controllers/VehicleTypeController.cs

```diff
@@ -23,21 +23,21 @@ IODCrudService<VehicleType> odService
         }
 
         [HttpGet("{id}")]
-        [Authorize(Roles = "ADMIN,ENGINEER,CUSTOMER,ANALYST,WORKER")]
+        [Authorize(Roles = "ADMIN,ANALYST")]
```

Comment:
How ENGINEER & CUSTOMER & WORKER should access to this endpoint?

Reply:
Resolve